openmeetings-util/src/main/java/org/apache/openmeetings/util/logging/PrometheusUtil.java

```diff
@@ -0,0 +1,35 @@
+/*
+
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License") +  you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.openmeetings.util.logging;
+
+import io.prometheus.client.Histogram;
+
+public class PrometheusUtil {
+
+	private static Histogram histogram = Histogram.build() //
```

Comment:
these `//` seems to be useless

Reply:
Those are quotes to prevent outformatting. You usually put them at the end, so that if you highlight all the lines and press CTRL-SHIFT-F Eclipse (or other editors) will reformat. And in order to force a line break you add those // at the end.